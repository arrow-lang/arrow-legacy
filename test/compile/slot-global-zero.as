let a: int;
let b: float;
let c: bool;
let m: (int, float, bool);
