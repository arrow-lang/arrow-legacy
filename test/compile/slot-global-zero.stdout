@a = global i32 0
@b = global double 0.000000e+00
@c = global i1 false
@m = global { i32, double, i1 } zeroinitializer

define void @"slot-global-zero.@init"() {
  ret void
}

define i32 @main(i32, i8**, i8**) {
  call void @"slot-global-zero.@init"()
  ret i32 0
}
